code/go/0chain.net/blobbercore/challenge/worker.go

```diff
@@ -30,6 +37,8 @@ func startProcessAccepted(ctx context.Context) {
 		select {
 		case <-ctx.Done():
 			return
+		case next := <-waitToProcess:
```

Comment:
do we still needs to be called from `processAccepted` ?

Reply:
@sum-it08  Yes. it is keep challenge will be lost to process even blobber crashes and restarts.